src/interfaces/util.py

```diff
@@ -0,0 +1,39 @@
+"""
+
+    This file contains all helper function for ui aspect implementation.
+    Any functional helper function for interactive component can be found at another util.py file.
+
+"""
+
+import os
+
+from PyQt5.QtWidgets import QLineEdit
+
+
+"""
+    Find a file in a given path string
+    reference: https://github.com/yahoo/TensorFlowOnSpark/blob/e2f5cc45f95812d163e75b6ddb9c4661261d3bb0/tensorflowonspark/util.py#L57
+
+    :param path: base path of the file
+    :param file: the name of the file, include postfix
+    :return: if exist file, return the path of the file, otherwise, return False
+"""
+def load_path(path, file):
```

Comment:
`file` is shadowing the `file` builtin which doesn't cause problems here but could potentially if we use it in the future. It's *pythonic* to avoid this and prefix the name with an _ like so: `def load_path(path, _file):`.

Reply:
fixed